src/company_cmd.cpp

```diff
@@ -182,7 +182,7 @@ void InvalidateCompanyWindows(const Company *company)
 {
 	CompanyID cid = company->index;
 
-	if (cid == _local_company) SetWindowDirty(WC_STATUS_BAR, 0);
+	if (cid == _local_company) SetWindowWidgetDirty(WC_STATUS_BAR, 0, 2);
```

Comment:
should this be `WID_S_RIGHT` instead of `2`? (honest question, magic numbers scare me, but I don't know this window system good enough)

Reply:
It could be. The only other call like this also uses the magic number though, in OnNewDay().

Reply:
Yeah, I think that should be `WID_S_LEFT`. Maybe good to fix them both?